File: pasien_app/lib/models/pasien.dart
class Pasien {
  final int id;
  final String noRm;
  final String namaPasien;
  final String jenisKelamin;
  final int umur;
  final String alamat;
  final String noHp;
  final String diagnosis;
  final String tanggalLahir;

  Pasien({
    required this.id,
    required this.noRm,
    required this.namaPasien,
    required this.jenisKelamin,
    required this.umur,
    required this.alamat,
    required this.noHp,
    required this.diagnosis,
    required this.tanggalLahir,
  });

  factory Pasien.fromJson(Map<String, dynamic> json) {
    return Pasien(
      id: json['id'],
      noRm: json['no_rm'] ?? '',
      namaPasien: json['nama_pasien'] ?? '',
      jenisKelamin: json['jenis_kelamin'] ?? 'L',
      umur: json['umur'] ?? 0,
      alamat: json['alamat'] ?? '',
      noHp: json['no_hp'] ?? '',
      diagnosis: json['diagnosis'] ?? '',
      tanggalLahir: json['tanggal_lahir'] ?? '',
    );
  }
}

File: backend_data_pasien/app/Models/Pasien.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Pasien extends Model
{
    use HasFactory;

    protected $table = 'pasiens';

    protected $fillable = [
        'no_rm',
        'nama_pasien',
        'jenis_kelamin',
        'tanggal_lahir',
        'umur',
        'alamat',
        'no_hp',
        'keluhan',
        'diagnosis'
    ];
}
